### DISCUSSION TITLE
Suppress Run Failed notifications for 'review time expired' workflow runs

### DISCUSSION THREAD

**Discussion by deadlydog** [community/community#121490] [Actions]
### Select Topic Area

Product Feedback

### Body

We use Environment "Required reviewers" to create approvals that block versions from being automatically deployed to production. For versions that we don't want deployed to prod, we just leave them and they expire after 30 days, which then results in an email notification of the failed run. This results in a lot of spam email notifications when we only promote 1 out of every 10 deployments to production. It would be awesome if there was a way to suppress notifications for workflow runs that failed because their review time expired.

**Comment by kkpkjadu** [comment]
ok

**Comment by qtips** [comment]
+1 Have the same issue. Please do share if anyone has a workaround. I was thinking of adding `if: ${{ job.status == 'cancelled' }}` and make the job successful with `continue-on-error: true` but I think that only works with jobs cancelled manually, and it will also ignore other types of errors, which I don't want.
